spec/registry.md

```diff
@@ -1109,6 +1109,31 @@ the functions `:datetime` and `:time`:
   - `true`
   - `false`
 
+The following _option_ and its values will be REQUIRED to be available on
+the functions `:datetime`, `:date`, and `:time`.
+This option currently has a Maturity Level of **Proposed**.
+
+- `timezone`
+  - A valid time zone identifier
+    (see [BCP175](https://www.rfc-editor.org/bpc/bpc175))
+  - `local`
+  - `UTC`
+  
+A time zone identifier is well-formed if it matches `tzId` in the following ABNF:
+```abnf
+tzId   = tzPath / tzEtc
+tzPath = tzPart 1*("/" tzPart)
+tzPart = tzWord *("_" tzWord)
+tzWord = (%x41-5A) *(%x61-7A) ; Uppercase ASCII letter followed by lowercase letters
+tzEtc  = ("Etc/" ("UTC" / "GMT" (("+" / "-") 1*2DIGIT))
+```
+  
+> [!NOTE]
+> The value `local` permits a _message_ to convert a date/time value
+> into a [floating](https://www.w3.org/TR/timezone/#floating) time value
+> (sometimes called a _plain_ or _local_ time value) by removing
+> the association with a specific time zone.
```

Comment:
I find this very confusing. From the name, my presumption would be that "local" means the formatter's local system time, but that's a time with an explicit timezone, while this note is indicating that it ought to mean a datetime not associated with any timezone.

With something like a literal date as operand, I presume that these would give the same results, but if the operand is an implementation-defined datetime with a timezone different from the local system time, it's not at all clear what date might get formatted.

I would find it very useful to have an example provided for a use case where `timeZone=local` would be required to get desirable behaviour, as opposed to just leaving out the option entirely.

Also, just for clarity, are there any use cases for floating datetimes that include anything more precise than a day being formatted?